### ISSUE TITLE
WKWebViewController

### ISSUE THREAD

**Issue by onato** [phonegap/phonegap-webview-ios#4]
Could you also add a podspec for https://github.com/apache/cordova-plugin-wkwebview-engine?

**Comment by onato** [comment]
```
{
  "name": "cordova-plugin-wkwebview-engine",
  "version": "1.0.1",
  "summary": "Cordova WKWebView Engine Plugin",
  "description": "The WKWebView Engine plugin allows you to use a WKWebView instead of a UIWebView.",
  "homepage": "https://github.com/apache/cordova-plugin-wkwebview-engine",
  "license": {
    "type": "Apache License, Version 2.0",
    "text": "    \"type\": \"Apache License, Version 2.0\",\n    \"text\": \"      Licensed under the Apache License, Version 2.0 (the \"License\");\n      you may not use this file except in compliance with the License.\n      You may obtain a copy of the License at\n      \n      http://www.apache.org/licenses/LICENSE-2.0\n      \n      Unless required by applicable law or agreed to in writing, software\n      distributed under the License is distributed on an \"AS IS\" BASIS,\n      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n      See the License for the specific language governing permissions and\n      limitations under the License.\n\"\n"
  },
  "authors": {
    "The Adobe PhoneGap Team": ""
  },
  "platforms": {
    "ios": "8.0"
  },
  "source": {
    "git": "https://github.com/apache/cordova-plugin-wkwebview-engine.git",
    "tag": "1.0.1"
  },
  "source_files": "src/ios/*.{h,m}",
  "requires_arc": true,
  "dependencies": {
    "Cordova": [
      ">= 3.0.0"
    ]
  }
}
```

**Comment by onato** [comment]
I see that **cordova-plugin-wkwebview-engine** wants Cordova 4. I would really like to get this going and am happy to help. Is there a roadmap for getting the pods updated to the latest version of Cordova?

**Comment by shazron** [comment]
Thanks @onato! Sending a pull request with your updates will speed things up. Currently we are off for our winter break right now and won't seriously review things until the new year.

The plugins themselves (save the wkwebview plugin) should be backwards compatible for any Cordova version.

**Comment by hollyschinsky** [comment]
Hi @onato! I'm going to work on getting the support for cordova-ios 4 out asap and will look at creating a podspec for the cordova-plugin-wkwebview-engine next. Cheers,
Holly

**Comment by hollyschinsky** [comment]
Hi @onato! 

I put out an updated [Cordova CocoaPod](https://cocoapods.org/pods/cordova) supporting 4.0.1 today and I also added a new
[cordova-plugin-wkwebview-engine](https://cocoapods.org/pods/cordova-plugin-wkwebview-engine) podspec. 

This sample template project has also been updated to use the latest versions of everything and defaults to `WKWebView`. I've tested it out with both Objective-C and Swift projects and it's all working.

Hope this helps!
Holly
